DigitalLearningSolutions.Data.Tests/Services/UserServiceTests.cs

```diff
@@ -137,5 +139,89 @@ public void GetAvailableCentres_returns_centres_correctly_ordered()
             // Then
             Assert.That(resultIdOrder.SequenceEqual(expectedIdOrder));
         }
+
+        [Test]
+        public void TryUpdateUsers_with_null_delegate_only_updates_admin()
+        {
+            // Given
+            var adminUser = UserTestHelper.GetDefaultAdminUser();
+            string password = "password";
+            A.CallTo(() => loginService.VerifyUsers(password, adminUser, A<List<DelegateUser>>._))
+                .Returns((adminUser, new List<DelegateUser>()));
+            A.CallTo(() => userDataService.UpdateAdminUser(adminUser)).DoesNothing();
+
+            // When
+            var result = userService.TryUpdateUsers(adminUser, null, password);
+
+            // Then
+            result.Should().BeTrue();
+            A.CallTo(() => userDataService.UpdateAdminUser(adminUser))
+                .MustHaveHappened();
```

Comment:
The name of the test says "only updates admin", but there's no check for whether the delegate user is updated or not. Would it be worth adding a check that UpdateDelegateUser is never called?